fantasy-docs.md

```diff
@@ -362,6 +393,37 @@ If your function accepts 0 parameters, then it can be visualized easily. But how
 
 This is a really powerful way to let consumers customize the goods you've designed before buying or manufacturing them. For example, you might put a design for a cool 3D printed office chair on thingiverse.com which has a function `main(name: Text)`. This function describes a chair with the given name embossed into the back. When a consumer wants to 3D print it, the 3D printing service will let them input their desired name, view the chair with that name embossed, and then order it.
 
+## Tests
+Functions are marked as tests using the `#[test]` attribute. They're run via the KittyCAD CLI or by the test runner built into the KittyCAD modeling app.
+
+```kcl
+#[test]
+division_by_1_doesnt_change_number() =
+    assert_eq(4/1, 4)
+
+#[test]
+division_by_10() =
+let
+    expected = 10;
+    actual = 100/10;
+in assert_eq(expected, actual)
+```
+The `assert_eq` function will fail the test if the arguments aren't equal. There are similar functions like `assert()` which just checks if its argument is true, or `assert_ne()` which asserts the two are not equal. Tests are run in parallel, because there's no way for two tests to interfere with each other. Test functions cannot take parameters, nor can they return values. If you want to automate tests, consider using smaller functions.
```

Comment:
>If you want to automate tests, consider using smaller functions.

I didn't quite get this sentence. Is it preemptive advice for readers who might not like the constraints of the testing attribute?

Reply:
OK, I changed this sentence.